File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.util.*;
import java.io.*;

public class Main {

	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		int N = Integer.parseInt(br.readLine());
		
		int[] nums = new int[N];
		for(int i = 0; i < N; i++) {
			nums[i] = Integer.parseInt(br.readLine());
		}
		
		Arrays.sort(nums);
		
		StringBuilder sb = new StringBuilder();
		for(int num : nums) {
			sb.append(num).append('\n');
		}
		
		System.out.print(sb);
	}

}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.kt
fun main() {
    val br = System.`in`.bufferedReader()
    val N = br.readLine().toInt()
    val arr = IntArray(N)
    repeat(N) {
        arr[it] = br.readLine().toInt()
    }
    arr.sort()
    val sb = StringBuilder()
    arr.forEach {
        sb.append(it).append("\n")
    }
    println(sb)
    br.close()
}